components/pkg-export-kubernetes/src/manifestjson.rs

```diff
@@ -24,12 +24,24 @@ use manifest::Manifest;
 const MANIFESTFILE: &'static str = include_str!("../defaults/KubernetesManifest.hbs");
 const BINDFILE: &'static str = include_str!("../defaults/KubernetesBind.hbs");
 
+/// Represents the [`Manifest`] in JSON format. This is an intermediate type that can be converted
+/// to the final manifest YAML file content, ready for consumption by your Kubernetes cluster.
+///
+/// The reason for the existance of this intermediate type is to allow users of this crate to be
+/// able to modify the JSON before converting it into the final manifest string.
```

Comment:
Nit: here you say "converted into"  but above "converted to". Intentional?

Reply:
not intentional but I think both are correct but I won't argue against consistency. :)